Cargo.toml

```diff
@@ -139,6 +139,7 @@ wildcard_dependencies             = "warn"
 zero_sized_map_values             = "warn"
 
 [workspace.lints.rustdoc]
+broken_intra_doc_links   = "warn"
```

Comment:
Here we enable lints that are `allow` by default because there tons of cool and useful lints in rustc, clippy and rustdoc that aren't enabled by default due to various reasons.

This lint, however, is already [`warn` by default](https://doc.rust-lang.org/rustdoc/lints.html#broken_intra_doc_links), so I don't think we need to list it here explicitly.

Reply:
Sorry, missed that part of the docs. Fixed.